pkg/reconciler/revision/queueing_test.go

```diff
@@ -141,6 +144,13 @@ func newTestController(t *testing.T) (
 	*configmap.ManualWatcher) {
 
 	ctx, cancel, informers := SetupFakeContextWithCancel(t)
```

Comment:
Why is the Secret informer not setup through this?

Reply:
@dgerd I think I will need to setup for fake informers for k8s core v1 but I don't think [this](https://github.com/knative/serving/pull/5760/files#diff-5345a009f642e55fc0d5b16d4f5d6f9bR59) is the right way to do it. What else am I missing?

Reply:
I think we need to update `update-codegen.sh` for k8s api injection. I will create an issue and a followup PR for it.